src/components/MessageList/MessageList.js

```diff
@@ -121,91 +121,155 @@ const MessageList = ({ messages, handleGoBack }) => {
                       {format(new Date(msg.ts), 'MMMM d, yyyy')}
                     </MessageDivider>
                   )}
-                  <Message.Header>
-                    <Message.Name>{msg.u?.name}</Message.Name>
-                    <Message.Username>@{msg.u.username}</Message.Username>
-                    <Message.Timestamp>
-                      {format(new Date(msg.ts), 'h:mm a')}
-                    </Message.Timestamp>
-                    {msg.editedAt && (
-                      <Icon mie="x4" opacity={0.5} name="edit" size="x16" />
-                    )}
-                  </Message.Header>
-                  <Message.Body>
-                    {msg.attachments && msg.attachments.length > 0 ? (
-                      <Attachments attachments={msg.attachments} />
-                    ) : (
-                      <Markdown body={msg.msg} />
-                    )}
-                  </Message.Body>
-                  <MessageReactions>
-                    {msg.reactions &&
-                      serializeReactions(msg.reactions).map((reaction) => (
-                        <MessageReactions.Reaction
-                          key={reaction.name}
-                          mine={isSameUser(reaction, authenticatedUserUsername)}
-                          onClick={() =>
-                            handleEmojiClick(
-                              reaction,
-                              msg,
-                              !isSameUser(reaction, authenticatedUserUsername)
-                            )
-                          }
-                        >
-                          <Markdown body={reaction.name} />
-                          <p>{reaction.count}</p>
-                        </MessageReactions.Reaction>
-                      ))}
-                  </MessageReactions>
+                  {msg.t === undefined ? (
+                    <>
```

Comment:
I don't think we need this fragment

Reply:
I think this fragment is necessary, without this fragment we will not be able to wrap up the children components under parent component. Or is there any way to wrap the components without using this fragment? If there is any way, then please elaborate me a little bit so that I could implement that. Thanks!

Reply:
Ahh sorry this is not the fragment that I was talking about,


Line 170
```
   <>
                      {msg.t === 'ul' && (
                        <Message.Header>
                          <Message.Name>@{msg.u.username} </Message.Name>
                          <Message.Username>
                    ....
```